tests/dft-vv10/input.dat

```diff
@@ -17,7 +17,28 @@ set DFT_VV10_RADIAL_POINTS 20
 set E_CONVERGENCE 1.e-12
 set D_CONVERGENCE 1.e-10
 
+
 scf_e, scf_wfn = energy("VV10", return_wfn=True)
 
 for k, v in bench.items():                        # TEST
     compare_values(v, psi4.get_variable(k), 9, k) # TEST
+
+
+scf_e, scf_wfn = energy("BLYP-NL", return_wfn=True)
+compare_values(0.032852665973,psi4.get_variable('DFT VV10 ENERGY') , 6, 'BLYP-NL dashtest') # TEST
+scf_nl=psi4.get_variable('TOTAL SCF ENERGY')
+
+set dft_dispersion_parameters [5.0]
+scf_e, scf_wfn = energy("blyp-nl", return_wfn=True)
+0.0238582087067217
+compare_values(0.0238582087067217,psi4.get_variable('DFT VV10 ENERGY') , 6, 'BLYP-NL custom b ') # TEST
+
+set DFT_VV10_B  4.0
+scf_e, scf_wfn = energy("BLYP", return_wfn=True)
+compare_values(0.032852665973,psi4.get_variable('DFT VV10 ENERGY') , 6, 'BLYP + custom VV10 ENERGY') # TEST
+
+set DFT_VV10_POSTSCF true
+scf_e, scf_wfn = energy("BLYP-NL", return_wfn=True)
+compare_values(0.032852665973,psi4.get_variable('DFT VV10 ENERGY') , 6, 'BLYP-NL postscf') # TEST
```

Comment:
This value looks identical to the self consistent one? Can you also add spaces after commas here.

Reply:
yes, I used the self consistent value to check if the deviation to the post-scf one is small, which is perhaps not in the spirit these tests are to be used.